### PR TITLE
added documentation link for connecting to socket

### PR DESCRIPTION
**Pull request by dnkmmr69420**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ meson _build --prefix=/usr/local
 ninja -C _build install
 ```
 
+## Link to the guide to enable the podman socket
```
Comment (marhkb) [inline README.md:104]: 2 Things:

1. The heading should be briefer, like "Podman Socket Activation"
2. As every heading has an emoji, I suggest to use 🔌 emoji here.

File: README.md [hunk 2]
```diff
@@ -101,6 +101,10 @@ meson _build --prefix=/usr/local
...
+
+[Podman socket activation](https://github.com/containers/podman/blob/cea9340242f3f6cf41f20fb0b6239aa3db5decd6/docs/tutorials/socket_activation.md)
```
Comment (marhkb) [inline README.md:106]: I'd like to have a bit more context here. The user who reads it should get an understanding why this is important.
I suggest writing something similar as in the lightbulb popup


> To connect to the local Podman instance, the systemd socket must be enabled. You can enable it permanently by issuing the following command:
> ```
> systemctl --user enable --now podman.socket
> ```
> Visit the <a href="https://github.com/containers/podman/blob/cea9340242f3f6cf41f20fb0b6239aa3db5decd6/docs/tutorials/socket_activation.md">official documentation</a> for more information.

**Review by marhkb — CHANGES_REQUESTED** [review]

**Comment by marhkb** [comment]
Thanks. This information is indeed important to have. I requested a few things to be changed.
Could you also change your commit to
```
chore(readme): Add socket activation instructions
```

**Comment by dnkmmr69420** [comment]
made a new pull request with those changes

**Pull request closed without merging**